config/initializers/config.rb

```diff
@@ -1,4 +1,5 @@
-SIGN_URL         = 'http://nzsl-assets.vuw.ac.nz/dnzsl/freelex/publicsearch'.freeze
-ASSET_URL        = 'http://nzsl-assets.vuw.ac.nz/dnzsl/freelex/assets/'.freeze
-AUTOCOMPLETE_URL = 'http://nzsl-assets.vuw.ac.nz/dnzsl/ncbin/public_search_lookup'.freeze
+SIGN_URL         = 'http://130.195.21.12/dnzsl/freelex/publicsearch'.freeze
+FREELEX_HOST     = 'nzsl-assets.vuw.ac.nz'.freeze
+ASSET_URL        = 'http://130.195.21.12/dnzsl/freelex/assets/'.freeze
+AUTOCOMPLETE_URL = 'http://130.195.21.12/dnzsl/ncbin/public_search_lookup'.freeze
```

Comment:
oh interesting!

The Freelex dev is on my team at work - i can ask for a DNS entry if you agree that looks nicer.